tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace/transformation_tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Any, Optional, Sequence, Union, overload, Literal, Generator
+
+import pytest
+import dace
+import copy
+import numpy as np
+from dace.sdfg import nodes as dace_nodes
+from dace.transformation import dataflow as dace_dataflow
+
+from gt4py.next import common as gtx_common
+from gt4py.next.program_processors.runners.dace_fieldview import (
+    transformations as gtx_transformations,
+)
+
+
+@pytest.fixture(autouse=True)
+def set_dace_settings() -> Generator[None, None, None]:
+    """Sets the common DaCe settings for the tests.
+
+    The function will modify the following settings:
```

Comment:
Please add a comment mentioning that because of the `autouse=True` option, this fixture will be automatically used by all tests inside this folder (and subfolders). To be honest, I'm not really sure if I like this implicit/magic behavior instead of  renaming the fixture to just `dace_config` and making it clear that it should be explicit called by all tests. Any other opinions about it @edopao @havogt ?

Reply:
@egparedes I do not understand what I should put in the comment?